bowling/src/lane.cpp

```diff
@@ -1,5 +1,34 @@
 #include "lane.hpp"
 
-Lane::Lane() {}
+#include <filesystem>
+
+// private
+
+void Lane::parseFile(std::string fileName)
+{
+    std::string fileExtension = std::filesystem::path(fileName).extension();
+    std::size_t foundExtensionIndex = fileName.find(fileExtension);
+
+    if (foundExtensionIndex != std::string::npos) {
+        fileName.erase(foundExtensionIndex);
+    }
+
+    this->laneName_ = fileName;
```

Comment:
general for all PR - there is no need to use `this` pointer if you work on class members from the class method. Simple:
`laneName_ = fileName;`
will work

Reply:
its one of my habits, for me its easier do understand :)